packages/rum-core/src/domain/view/viewMetrics/trackLoadingTime.spec.ts

```diff
@@ -36,114 +47,85 @@ describe('trackLoadingTime', () => {
   })
 
   it('should have an undefined loading time if there is no activity on a route change', () => {
-    const { clock } = setupBuilder.build()
-    const { getViewUpdate, getViewUpdateCount, startView } = viewTest
-
-    startView()
-    clock.tick(THROTTLE_VIEW_UPDATE_PERIOD)
+    setupBuilder.build()
 
-    expect(getViewUpdateCount()).toEqual(3)
-    expect(getViewUpdate(2).commonViewMetrics.loadingTime).toBeUndefined()
+    expect(loadingTimeCallback).not.toHaveBeenCalled()
   })
 
   it('should have a loading time equal to the activity time if there is a unique activity on a route change', () => {
     const { domMutationObservable, clock } = setupBuilder.build()
-    const { getViewUpdate, startView } = viewTest
 
-    startView()
     clock.tick(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY)
     domMutationObservable.notify()
     clock.tick(AFTER_PAGE_ACTIVITY_END_DELAY)
-    clock.tick(THROTTLE_VIEW_UPDATE_PERIOD)
 
-    expect(getViewUpdate(3).commonViewMetrics.loadingTime).toEqual(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY)
+    expect(loadingTimeCallback).toHaveBeenCalledTimes(1)
+    expect(loadingTimeCallback).toHaveBeenCalledWith(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY)
   })
 
   it('should use loadEventEnd for initial view when having no activity', () => {
+    loadType = ViewLoadingType.INITIAL_LOAD
     const { lifeCycle, clock } = setupBuilder.build()
-    const { getViewUpdate, getViewUpdateCount } = viewTest
 
-    expect(getViewUpdateCount()).toEqual(1)
     const entry = createPerformanceEntry(RumPerformanceEntryType.NAVIGATION)
     lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, [entry])
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏remove this useless `lifeCycle.notify`